Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -264,11 +258,7 @@ private static DeployType getDeployTypeByWebContainer(com.microsoft.azure.toolki
             type = AzureOperation.Type.SERVICE
     )
     public void updateDeploymentSlotAppSettings(final IWebAppDeploymentSlot slot, final Map<String, String> toUpdate) {
```

Comment:
can this method be removed?

Reply:
This could be removed after we migrate to tasks in app service library, will be done later